src/App.css

```diff
@@ -271,3 +271,16 @@ span {
 .btn:hover {
 	transform: scale(1.05);
 }
+
+
+.copied-clipboard {
+    position: absolute;
+    bottom: 1em;
+    right: 2em;
+    font-size: 0.9rem;
```

Comment:
Suggestion : if computed value is less than 10 px , then use px

Reply:
done